src/js/userAccount.jsx

```diff
@@ -12,237 +11,227 @@ import Modal from "./components/Modal";
-            {this.renderField(users?.email, "email", "email", true)}
-            {this.renderField(users?.fullName, "text", "fullName")}
-            {this.renderField(users?.institution, "text", "institution")}
-            {this.renderSelect(
-              users?.sector,
-              [
-                { value: "academic", label: users?.academic },
-                { value: "government", label: users?.government },
-                { value: "ngo", label: users?.ngo },
-              ],
-              "sector"
-            )}
-            <div style={{ display: "flex", justifyContent: "space-between", alignItems: "center" }}>
-              <span style={{ color: "red" }}>{users?.allRequired}</span>
-              <div style={{ display: "flex", marginTop: "0.5rem" }}>
-                <Button
-                  onClick={() => {
-                    this.showAlert({
-                      body: users?.logOutBody,
-                      closeText: users?.cancel,
-                      confirmText: users?.confirmText,
-                      onConfirm: () => window.location.assign("/logout"),
-                      title: users?.logout,
-                    });
-                  }}
-                  extraStyle={{ marginRight: "0.5rem" }}
-                >
-                  {users?.logout}
-                </Button>
-                <Button
-                  onClick={() => {
-                    this.showAlert({
-                      body: users?.updateBody,
-                      closeText: users?.cancel,
-                      confirmText: users?.confirmText,
-                      onConfirm: () => this.updateUser(),
-                      title: users?.userAccountTitle,
-                    });
-                  }}
-                >
-                  {users?.save}
-                </Button>
-              </div>
-            </div>
-          </AccountForm>
-          {this.state.messageBox && (
-            <Modal {...this.state.messageBox} onClose={() => this.setState({ messageBox: null })}>
-              <p>{this.state.messageBox.body}</p>
-            </Modal>
-          )}
-        </PageContainer>
-      </ThemeProvider>
-    );
-  }
+          </div>
+        </AccountForm>
+        {messageBox && (
+          <Modal {...messageBox} onClose={() => setMessageBox(null)}>
+            <p>{messageBox.body}</p>
+          </Modal>
+        )}
+      </PageContainer>
+    </ThemeProvider>
+  );
 }
 
-UserAccount.contextType = MainContext;
-
 export function pageInit(args) {
   ReactDOM.render(
-    <PageLayout
-      role={args.role}
-      userLang={args.userLang}
-      username={args.username}
-      showSearch={false}
-    >
-      <UserAccount />
-    </PageLayout>,
+    <Suspense fallback="">
+      <PageLayout
+        role={args.role}
+        username={args.username}
+        mapboxToken={args.mapboxToken}
+        mapquestKey={args.mapquestKey}
+        versionDeployed={args.versionDeployed}
+        showSearch={true}
```

Comment:
This should be `showSearch={false}`. I also don't think we need to provide the `mapboxToken` or `mapquestKey` here